packages/generator/tests/acceptance/test_streaming_measurement_client.py

```diff
@@ -0,0 +1,95 @@
+import importlib.util
+import pathlib
+from types import ModuleType
+from typing import Generator
+
+import pytest
+from ni_measurement_plugin_sdk_service.measurement.service import MeasurementService
+
+from ni_measurement_plugin_sdk_generator.client import create_client
+from tests.utilities.discovery_service_process import DiscoveryServiceProcess
+from tests.utilities.measurements import streaming_data_measurement
```

Comment:
I didn't add import-paths, instead, used py.typed for that and I was re-using the assets. 

Now,
- I have removed the py.typed file and added the measurements required for generator and deleted the measurement measurement added in packages/service/tests/assets.
- I have also added __init__.py in  packages/generator/tests.

Reply:
You did add import paths. You just didn't know it.
- `pyproject.toml` is installing `ni-measurement-plugin-sdk-service` from `../../packages/service`.
- Each package is using "flat layout": https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/
- Because `ni-measurement-plugin-sdk-service` uses flat layout, this adds `../../packages/service` to the import path. (If it used src layout, I think it would add `../../packages/service/src`.)
- The tests are also in this directory, so they also get added to the import path.

This will break when we create the next prerelease and comment out the ../../ stuff in `pyproject.toml`, because it will use `.venv/lib/site-packages/ni-measurement-plugin-sdk-service` rather than adding `../../packages/service` to the import path.

I agree with adding test assets/utilities to `packages/generator/tests` and removing `py.typed` from the tests directory.

Reply:
This is also because you set `develop = true`. Removing this makes the tests behave like they should:

```
PS D:\dev\measurement-plugin-python\packages\generator> poetry run pytest -v
ImportError while loading conftest 'D:\dev\measurement-plugin-python\packages\generator\tests\conftest.py'.
tests\conftest.py:11: in <module>
    from tests.utilities.discovery_service_process import DiscoveryServiceProcess
E   ModuleNotFoundError: No module named 'tests.utilities'
```

Reply:
Sure, I got it. Thanks. I have removed `develop = true`.